content/en/docs/reference/glossary/events.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Events
+id: k8s-events
+date: 2019-08-30
+full_link: /docs/concepts/cluster-administration/events/
+short_description: >
+  Events are objects that provide insight into what is happening inside a {{< glossary_tooltip text="cluster" term_id="cluster" >}}.
```

Comment:
```suggestion
   Objects that record occurrences or actions inside your cluster.
```
[In my opinion], `short_description` fields work best without tooltips in them.

Reply:
removed tooltip